www/source/docs/contribute-help-build.html.md

```diff
@@ -12,7 +12,7 @@ The remainder of this document describes some of the areas of the product that c
 
 ## More Topologies
 
-Habitat ships on day one with [two topologies](/docs/run-packages-topologies): standalone, and leader-follower. There are undoubtedly more that are needed in order to represent the semantics of real systems.
```

Comment:
... I think we are in agreement. With only two items left in the list, that's why I removed it. Or am I misunderstanding your comment?